docs/how-to/nat-configuration.md

```diff
@@ -1,32 +1,136 @@
 
-While the NAT is usually transparent for outgoing connections, listening for incoming connections requires some configuration. The router listens on a single public IP address, but any number of machines on the internal network could handle the request. To serve requests, your router must be configured to send certain traffic to a specific machine. You can configure your router to allow these connections by either of the following sections.
+While NATs are generally transparent for outgoing connections, listening for incoming connections requires some configuration. The router listens on a single public IP address, but any number of machines on the internal network could handle the request. To serve requests, your router must be configured to send certain traffic to a specific machine. 
+
+## Configuration Options
 
-## IPv6
+The appropriate configuration option for your router depends on your specific setup:
+
+- If your router supports them, [enable IPv6](#enable-ipv6) or [enable UPnP](#enable-upnp) to solve most connection issues
+- If IPv6 or UPnP are not available, [enable manual port forwarding](#enable-manual-port-forwarding)
+
+### Enable IPv6
 
 If your router and internet service provider (ISP) support IPv6, enabling it will mitigate some connection issues. Most modern routers have a single option to enable IPv6. We are unable to give detailed information about each router's settings and preferences here. Search your router manufacturer's website for _IPv6_ for information on how to enable it.
 
-## UPnP
+### Enable UPnP
 
 If your router supports UPnP, IPFS will attempt to automatically allow inbound traffic to access your local content. Some home routers may need to be configured to explicitly enable UPnP. We are unable to give detailed information about each router's settings and preferences here. Search your router manufacturer's website for _UPnP_.
 
-## Port forwarding
+<!--
+### Enable DCUtR Holepunching 
+
+Not sure what to put here
+
+Is there a recommended repo or libp2p docs link that we can link to here that explains how to set this up for Kubo? 
+
+https://github.com/libp2p/specs/blob/master/relay/DCUtR.md doesn't seem right
+
+https://docs.libp2p.io/concepts/nat/hole-punching/#phase-ii-hole-punching? seems to be mostly conceptual
+
+https://blog.ipfs.tech/2022-01-20-libp2p-hole-punching/ seems to just explain the theory
+
+Maybe https://docs.rs/libp2p/latest/libp2p/tutorials/hole_punching/index.html and tell the reader that they can follow this tutorial to set up DCUtR Holepunching using Rust? Not specific to Kubo but seems like its the best we've got right now?
+!-->
```

Comment:
Dcutr is enabled by default, the user does not have to do anything.
I think we just need to explain what it is the draw backs and / or link to the blog post.

Some people have hole punchable nats so they see some success rate, however they would have better connectivity with a port forward.

Reply:
Alright, based the info off of kubo/docs and the blog post 
https://bafybeidbxtj5a5t6ashyyaq2j23gy6q2muga2yaftbkx2xxlgjkebmax6u.on.fleek.co/how-to/nat-configuration/#disable-dcutr-holepunching